.moban.dt/core-setup.py.jj2

```diff
@@ -0,0 +1,49 @@
+{% extends 'coala-setup.py.jj2'%}
+
+{%block local_import_block%}
```

Comment:
I don't see this block on the template which you are extending? i.e. https://github.com/jayvdb/coala-mobans/blob/master/templates/coala-setup.py.jj2